src/frontend/components/RedirectOnLoad/index.tsx

```diff
@@ -39,7 +38,7 @@ export const RedirectOnLoad = () => {
     return <Redirect push to={DOCUMENT_PLAYER_ROUTE()} />;
   }
 
-  if (ltiState === appState.INSTRUCTOR) {
+  if (appData.isEditable) {
     if (resource!.upload_state === uploadState.PENDING) {
       return (
         <Redirect
```

Comment:
Maybe the comment on line 54 should be updated to reflect `isEditable` instead of roles